doc/news/_preparation_next_release.md

```diff
@@ -155,6 +155,10 @@ The following section lists news about the [modules](https://www.libelektra.org/
 
 - We fixed an incorrect format specifier in a call to the `syslog` function. _(René Schwaiger)_
 
+### Cache
+
+- [cache](https://www.libelektra.org/plugins/cache) is a new global caching plugin. It uses [mmapstorage](https://www.libelektra.org/plugins/mmapstorage) as its storage backend and lazily stores keysets from previous ´kdbGet()´ calls. _(Mihael Pranjić)_
```

Comment:
Please highlight it and also put benchmarks here (does not need to be within this PR, though)